**Известно, что генеральная совокупность распределена нормально, со средним квадратическим отклонением, $\displaystyle\sigma=16$. Найти доверительный интервал для оценки математического ожидания с надежностью $0.95$, если выборочная средняя $M=80$, а объем выборки $n=256$.**

Поскольку нам известно среднее квадратическое отклонение генеральной совокупности $\sigma=16$, то для расчета $95\%$-го доверительного интервала, будем использовать $Z$-критерий. И границы доверительного интервала расчитаем по формуле:  
$\displaystyle\bar X\pm z_{\alpha/2}\cdot\frac{\sigma}{\sqrt n}$, где:  
$\bar X=M=80$-среднее значение выборки    
$z_{\alpha/2}$ табличное значение $Z$-критерия, и для $95\%$-го доверительного интервала $z_{\alpha/2}=1,96$  
$\sigma=16$-среднее квадратическое отклонение генеральной совокупности  
$n=256$-объем выборки  
Таким образом $95\%$-й доверительный интервал расчитаем как: $\left[80-1.96\cdot\frac{16}{\sqrt 256};80+1.96\cdot\frac{16}{\sqrt 256}\right]$ 

In [1]:
left=80-1.96*16/256**(1/2)
right=80+1.96*16/256**(1/2)
print(f'95%-й доверительный интервал для оценки мат. ожидания генеральной совокупности: [{left};{right}].')

95%-й доверительный интервал для оценки мат. ожидания генеральной совокупности: [78.04;81.96].


---  
Ответ:  
95%-й доверительный интервал для оценки мат. ожидания генеральной совокупности равен $\left[78.04;81.96\right]$  

---